readme.rst

```diff
@@ -1,25 +1,74 @@
 Documentation starter pack
 ==========================
 
-See the `Read the Docs at Canonical <https://library.canonical.com/documentation/read-the-docs>`_ and
-`How to publish documentation on Read the Docs <https://library.canonical.com/documentation/publish-on-read-the-docs>`_ guides for
-instructions on how to get started with Sphinx documentation.
-
-Then go through the following sections to use this starter pack to set up your docs repository.
+The Documentation starter pack contains tools and configuration options for
+building a documentation site with Sphinx. Its main features include:
+
+* bundled Sphinx theme, configuration, and extensions
+* build checks: links, spelling, inclusive language
+
+One strong point of the starter pack is its support for customisation that is
+layered on top of a core configuration - this applies to all the above
+features. There are also many other goodies included to enrich your doc set.
+
+.. toctree::
+   :hidden:
+
+   Setup script <setup-script>
+
+Contents
+--------
+
+This README is divided into two main parts: enable the starter pack or work
+with your documentation post-enablement.
+
+* `Enable the starter pack`_
+    + `Two supported scenarios`_
+        + `Standalone documentation repository`_
+        + `Documentation in a code repository`_
+    + `Build the documentation`_
+    + `Configure the documentation`_
+        + `Configure the header`_
+        + `Activate/deactivate feedback button`_
+        + `Configure included extensions`_
+        + `Add custom configuration`_
+    + `Change log`_
+* `Work with your documentation`_
+    + `Install prerequisite software`_
+    + `View the documentation`_
+    + `Local checks`_
+        + `Local build`_
+        + `Spelling check`_
+        + `Inclusive language check`_
+        + `Link check`_
+    + `Configure the spelling check`_
+    + `Customisation of inclusive language checks`_
+    + `Configure the link check`_
+    + `Add redirects`_
+    + `Other resources`_
```

Comment:
There's something wrong with the indentation here - some of the lines show up in bold (both in the doc output and in the GitHub rendering). I think there's too much indentation, which turns it into a definition list (inside of the bulleted list).